extensions/global-media-key/package.json

```diff
@@ -0,0 +1,68 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "global-media-key",
+  "title": "Media Key Emulate",
+  "description": "Emulate keyboard media keys press. Reassign hotkey for media keys.",
+  "icon": "extension_icon.png",
+  "author": "douo",
+  "categories": [
+    "System"
```

Comment:
Maybe you should add `Media` as a category too. What do you think?

Reply:
Sure!